games-arcade/rocksndiamonds/rocksndiamonds-3.3.1.2.recipe

```diff
@@ -59,5 +62,19 @@ INSTALL()
 	destdir=$appsDir/"Rocks'n'Diamonds"
 	mkdir -p $destdir
 	cp -r rocksndiamonds sounds graphics levels music $destdir
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local INTERNAL="`echo "$portVersion" | cut -d. -f4`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@INTERNAL@|$INTERNAL|" \
+		$portDir/additional-files/rocksndiamonds.rdef > rocksndiamonds.rdef
```

Comment:
Shall we replace
```
		$portDir/additional-files/rocksndiamonds.rdef > rocksndiamonds.rdef
```
by
```
		$portDir/additional-files/rocksndiamonds.rdef.in > rocksndiamonds.rdef
```

(I had attached this comment to the wrong line in my previous review, so I removed it and added a single message, on the right line.)